### ISSUE TITLE
Deleting a duplicated page causes Uncaught (in promise) undefined

### ISSUE THREAD

**Issue by GiantCrocodile** [getkirby/kirby#2712]
**To Reproduce**  
1. Create a page on root level called 'test123'
2. Click on the page to open it
3. Duplicate the page and just confirm
4. go back to previous page (e. g. by using the back button in your browser)
5. delete the previously page which was duplicated
6. the console shows an error

**Expected behavior**  
No error should happen in console.

**Screenshots**  
![grafik](https://user-images.githubusercontent.com/6271057/87551661-f9759d80-c6b0-11ea-8394-a64b82605fb1.png)

**Kirby Version**  
Kirby v3.4.0

**Console output**  
see above

**Desktop (please complete the following information):**  
latest FireFox on Win 10

**Additional context**
This is a multi lang. site with just 1 default language defined.

**Comment by afbora** [comment]
I tried deleting it in different ways, but I couldn't reproduce the issue. Works for me on Chrome and Firefox. Caches, plugins maybe? 🤔

**Comment by GiantCrocodile** [comment]
Caching can't be an issue here as my cache is deleted on every browser session. Maybe something did corrupt the Kirby 3 cache during the session?

Plugins:
- `editor` plug-in
- a quite unfinished comment plug-in:

index.php
```
<?php

Kirby::plugin('<removed>/comment', [
    'hooks' => [
        'kirbytags:after' => function ($text, $data, $options) {
            $session = kirby()->session();

            if ($location = $session->get('referer')) {
                if ($page = page(urldecode($location))) {
                    $title = $page->title();
                }
            }

            return Str::template($text, [
                'comment' => $title ?? '',
                'commentName'  => $session->get('commentName') ?? ''
            ]);
        }
    ],
]);
```

I did update the name and email of the user in the panel while I'm logged in with but not sure if that matters.

**Comment by afbora** [comment]
You still get the error without any plugins?

**Comment by GiantCrocodile** [comment]
I did move the two plugins outside the webserver's directory and the issue persists.

**Comment by afbora** [comment]
I really have no idea. So do you use any hooks?

**Comment by GiantCrocodile** [comment]
Did you delete the cloned page (the copy) or the page you did copy from (the original you did duplicate)?

config.yml:
```
return [
    'debug'  => true,
	'languages' => true,
	'markdown' => [
		'extra' => true
	],
	'api' => [
		'basicAuth' => true,
		'allowInsecure' => true
    ],
	'hooks' => [
		'page.create:after' => function ($page) {
			if($page->template()->name() == 'article') {
				$comment = $page->createChild([
                    'slug'     => 'comments',
                    'template' => 'comments',
                    'content'  => null,
					'draft'	   => false
                ]);
			$page->update(['Children' => Yaml::encode($page->childrenAndDrafts()->toArray())]);
			}
        }
	]
];
```

The hook will do nothing as the template used during duplication was NOT 'article' but a different template.

**Comment by afbora** [comment]
I tried both but no console output.

**Comment by afbora** [comment]
Did you try on starterkit or plainkit?

**Comment by GiantCrocodile** [comment]
@afbora I tried with starterkit and the issue doesn't occur. So I bet some update/bug in the past did break something in my setup. I did update this setup several times thus I wouldn't be surprised if I'm hit by some migration issue or some bug in update path. Is there any kind of files to check or some kind of files to delete as they are temporary/useless?

If you like to, I can zip you this project and send it via email to you as it contains no confidential data. Posting it here wouldn't be that good.

**Comment by afbora** [comment]
I have an idea about that: Fork `kirby` repo and clone to desktop. Replace your website `kirby` folder to cloned `kirby` folder. You will see changed files with CLI, GUI or GitHub Desktop.

**Comment by GiantCrocodile** [comment]
I did that and many changes are shown because it tells me that line endings will change from Linux/Windows one. What I did find out when I compare both master branches:

New files:
- `vendor\phpmailer\phpmailer\language\phpmailer.lang-am.php`)
- `vendor\zendframework\zend-escaper\src\Escaper.php`
- `vendor\zendframework\zend-escaper\src\Exception\ExceptionInterface.php`
- `vendor\zendframework\zend-escaper\src\Exception\InvalidArgumentException.php`
- `vendor\zendframework\zend-escaper\src\Exception\RuntimeException.php`

There are no files where source code differs like lines added/removed/changed. So I suspect that my Kirby 3 directory should be healthy although it has more files than the master branch (I'm surprised by that).

**Comment by afbora** [comment]
I'm closing this for now. If you experience the issue again or catch a hint, feel free to reopen the issue 👍

**Comment by GiantCrocodile** [comment]
@bastianallgeier Do you have an idea if this is related to the other issue? Any idea what to check?

**Comment by lukasbestle** [comment]
@GiantCrocodile The `kirby` directory doesn't contain dynamic content, you can completely replace it with a fresh one. You should also delete all files and directories in `media/panel`, `site/cache` and `site/sessions` as these files will be regenerated automatically.

What is the "other issue" you are referring to?

**Comment by GiantCrocodile** [comment]
@lukasbestle I will do replace the `kirby` dir and clean the other dirs.

I refer to #2713.

**Comment by lukasbestle** [comment]
No, #2713 was a bug that will be fixed in Kirby 3.4.1.
